wrappers/hnygrpc/grpc.go

```diff
@@ -72,16 +74,26 @@ func addFields(ctx context.Context, info *grpc.UnaryServerInfo, handler grpc.Una
 	span.AddField("handler.name", handlerName)
 	span.AddField("handler.method", info.FullMethod)
 
-	md, ok := metadata.FromIncomingContext(ctx)
+	pr, ok := peer.FromContext(ctx)
 	if ok {
-		if val, ok := md["content-type"]; ok {
-			span.AddField("request.content_type", val[0])
-		}
-		if val, ok := md[":authority"]; ok {
-			span.AddField("request.header.authority", val[0])
+		if pr.Addr != net.Addr(nil) {
+			span.AddField("request.remote_addr", pr.Addr.String())
```

Comment:
> But it's a "peer" not a remote.

I opted for naming parity with HTTP requests. `remote_addr` will be the IP and socket of actual HTTP/GRPC remote endpoint talking to this instrumented service. It could be the actual GRPC client or it could be the IP:port of a proxy shimmed in between.